src/asm/lexer.c

```diff
@@ -1031,10 +1038,11 @@ static void readLineContinuation(void)
 		} else if (c == '\r' || c == '\n') {
 			shiftChars(1);
 			if (c == '\r' && peek(0) == '\n')
+				/* Handle CRLF */
 				shiftChars(1);
 			if (!lexerState->expansions
```

Comment:
I would add a comment noting that characters must be shifted *before* `nextLine()`, since `shiftChars` updates the column count.

Reply:
Done.